### DISCUSSION TITLE
Not getting a label for toxic text prompt.

### DISCUSSION THREAD

**Discussion by VectorAnalytics** [GoogleCloudPlatform/generative-ai#61] [Q&A]
I'm working thru text_classification.ipynb.  In the section on toxicity detection, I can change the text and generate new "non-tonic" labels. But when I enter text that depicts racism or a text with a "bad" word, I don't get any response from the model. I was expecting to receive the "toxic" label. Do we not get any label for toxic text?

**Comment by polong-lin** [comment]
Hi @VectorAnalytics ,
The safety filter catches and blocks responses if the safety filter is triggered based on the contents of the prompt, which is likely what happened in your case. 
https://cloud.google.com/vertex-ai/docs/generative-ai/learn/responsible-ai#safety_filters_and_attributes
